src/components/ProjectList/listOfProjects.js

```diff
@@ -556,6 +556,20 @@ const projectList = [
     githubLink: 'https://github.com/moby/moby',
     description: 'The Moby project is a collaborative project for the container ecosystem to assemble container-based systems',
     tags: ['docker', 'containers', 'go']
+  },
+  {
+    name: 'ScyllaDb',
+    imageSrc: 'https://avatars1.githubusercontent.com/u/14364730?s=200&v=4',
+    githubLink: 'https://github.com/scylladb/scylla/blob/master/CONTRIBUTING.md',
```

Comment:
It will be more consistent with other projects if we point the GitHub link to the main repo or the org:
I prefer: https://github.com/scylladb/scylla

Reply:
Hi @sara-02 , addressed the review comment.